recipes/fiftyone/launch-jupyter.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Is this file used here? It doesn't seem like. We should not crate envs and delete folders at build time BTW. The test section will install the package and we can design this test, if it is really needed, to use that instead.

Reply:
This is not used in the build. It's more like a maintainer tool to test if the package actually works.

Reply:
Then you don't need this here, right?

Reply:
If I don't need to test updates later then no

CI green, LGTM, ship it 🚀

Reply:
> If I don't need to test updates later then no

Then let's remove it and the notebook. Having them here won't make it part of the package and you are not using them.

Reply:
Done